pyat/at/physics/radiation.py

```diff
@@ -3,11 +3,12 @@
 """
 from math import sin, cos, tan, sqrt, sinh, cosh, pi
 import numpy
-from typing import Tuple
+from typing import Tuple, Union
```

Comment:
Not related to the modifications, but for cleanup for python > 3.8 (use standard type for annotations):
```python
from __future__ import annotations
from math import sin, cos, tan, sqrt, sinh, cosh, pi
import numpy
from typing import Union
```
and replace `Tuple` with `tuple` everywhere.